app/config/config.local.neon.example

```diff
@@ -39,6 +39,8 @@ parameters:
     apiUrl: https://recodex.mff.cuni.cz:4000
     footerUrl: https://recodex.mff.cuni.cz
     from: "ReCodEx <noreply@example.com>"
+    #debugMode: true  # uncomment for debugging
```

Comment:
the comments in global config file and local are not the same... in this case they are important because debug mode is defined very differently from the regular use case...